src/main/java/com/iexec/worker/docker/DockerRegistryConfiguration.java

```diff
@@ -31,12 +33,25 @@
 @Slf4j
 @Configuration
 @ConfigurationProperties(prefix = "docker")
+@Getter
+@Setter
 public class DockerRegistryConfiguration {
 
-    @Setter
-    @Getter
     private List<RegistryCredentials> registries;
 
+    /**
+     * Min pull timeout expressed in minutes
+     */
+    @Positive
+    @Value("${docker.image.pull-timeout.min}")
+    private long minPullTimeout;
```

Comment:
Is it possible to use `Duration` instead of `long`?

Reply:
Sure, see https://github.com/iExecBlockchainComputing/iexec-worker/pull/442/commits/75c5434b0ec23721e8f55b178adac6eb4277a833.